### DISCUSSION TITLE
Docker

### DISCUSSION THREAD

**Discussion by bradgyton** [honzaap/Systemizer#41] [Ideas]
Would be great if this could be "dockerised" 

Might do it myself if i get some spare time

**Comment by bradgyton** [comment]
Threw this together. Needs abit of tidying but works a treat!

dockerfile

```dockerfile
# base image
FROM tarampampam/node

# set working directory
WORKDIR /app

# add `/app/node_modules/.bin` to $PATH
ENV PATH /app/node_modules/.bin:$PATH

# install and cache app dependencies
RUN git clone https://github.com/honzaap/Systemizer /app
RUN yarn

EXPOSE 4200

# start app
CMD ng serve --host 0.0.0.0 --port 4200 --disable-host-check
```

**Reply by honzaap** [reply]
It all seems good to me. You can create a pull request or I'll add it once I have time. It will probably need some small info in Wiki aswell like "How to run with docker" etc
